### DISCUSSION TITLE
Read only access to all repos by a single team

### DISCUSSION THREAD

**Discussion by engops-rob** [community/community#22573] [New to GitHub]
<p>Hi,</p>
<p>Beyond shell scripting I’m not a developer. I have, however been asked to solve a problem. </p>
<p>Our security team needs read-only access to all repos in our Org. <a href="https://help.github.com/articles/managing-team-access-to-an-organization-repository/" rel="nofollow noopener">Docs show me how to do this manually</a>, but it looks like I can only do it one repo at a time. </p>
<p>Is there a way to quickly give org-wide read-only access of _ <strong>all</strong> _ repos to a specific team?</p>
<p>Thanks, R</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/engops-rob">@engops-rob</a>,</p>
<p>Thank you for being here! One way to accomplish this would be to use the GitHub API: <a href="https://developer.github.com/v3/teams/#add-or-update-team-repository" rel="nofollow noopener">https://developer.github.com/v3/teams/#add-or-update-team-repository</a>. We have a board dedicated to <a href="/t5/GitHub-API-Development-and/bd-p/api">API topics</a> in case you run into any questions. I hope this helps!</p>
<p>Best,</p>
<p>Andrea</p>

**Reply by mewc** [reply]
second link doesn't work, and is that really the only way...very simple common requirement

**Comment by engops-rob** [comment]
<p>I’ve had the question in that forum for 5 weeks with no response, and at least one other person has a very similar request. Any thoughts how to get some activity on the thread?</p>

**Reply by RafaelBroseghini** [reply]
You could setup a personal access token and have a github workflow triggered by a cron job go through all repos in your org and assign read permissions to the security team.
